repoclosure/yum.conf

```diff
@@ -45,35 +45,35 @@ baseurl=https://yum.puppetlabs.com/el/7/PC1/x86_64/
 name=el7-puppet-6
 baseurl=https://yum.puppetlabs.com/puppet6/el/7/x86_64/
 
-[el7-foreman-rails-nightly]
-name=Foreman Rails nightly EL7
+[el7-foreman-rails-1.24]
+name=Foreman Rails 1.24 EL7
 baseurl=http://koji.katello.org/releases/yum/tfm-ror52/el7/x86_64/
```

Comment:
For 1.23 we've created a `foreman-rails-1.23` tag that branched off `tfm-ror52`. so this probably needs an update. I'd check with 1.23.